frontends/web/src/routes/buy/moonpay.tsx

```diff
@@ -85,33 +84,15 @@ class Moonpay extends Component<Props, State> {
     }, 200);
   };
 
-  private getAccount = (): IAccount | undefined => {
-    if (!this.props.accounts) {
-      return undefined;
-    }
-    return this.props.accounts.find(({ code }) => code === this.props.code);
-  };
-
-  private getCryptoName = (): string => {
-    const { t } = this.props;
-    const account = this.getAccount();
-    if (account) {
-      return isBitcoinOnly(account.coinCode) ? 'Bitcoin' : t('buy.info.crypto');
-    }
-    return t('buy.info.crypto');
-  };
-
-
   public render() {
     const { moonpay, t } = this.props;
     const { agreedTerms, height, iframeLoaded } = this.state;
-    const account = this.getAccount();
-
+    const account = getAccount(this.props.accounts, this.props.code);
     if (!account || moonpay.url === '') {
       return null;
     }
 
-    const name = this.getCryptoName();
+    const name = getCryptoName(this.props.t, account);
```

Comment:
Instead of passing  `t` function, how about using the 2nd arg to pass the cryptoName string, like this:

```js
getCryptoName(account, t('buy.info.crypto'));
```

(I know we pass t function in other places, that we should clean up one day)

Reply:
Nice, done!